src/main/java/co/lotc/lever/listener/FallDamageListener.java

```diff
@@ -0,0 +1,32 @@
+package co.lotc.lever.listener;
+
+import lombok.RequiredArgsConstructor;
+import org.bukkit.entity.Player;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.EventPriority;
+import org.bukkit.event.Listener;
+import org.bukkit.event.entity.EntityDamageEvent;
+import org.bukkit.event.player.PlayerQuitEvent;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.bukkit.event.entity.EntityDamageEvent.DamageCause.FALL;
+
+@RequiredArgsConstructor
+public class FallDamageListener implements Listener {
+	public static final Set<Player> negateFallDamage = new HashSet<>();
+
+	@EventHandler(ignoreCancelled = true, priority = EventPriority.HIGH)
```

Comment:
I'm not very familiar with EventPriority. I just arbitrarily picked HIGH. Everyone on the beta server has godmode, so I only did some testing with EventPriority.LOWEST and sent debug info to the player when they took fall damage :p